go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/onsi/ginkgo v1.14.1
 	github.com/onsi/gomega v1.10.2
 	github.com/stretchr/testify v1.5.1
+	go.uber.org/zap v1.15.0
```

Comment:
can  we use latest version ie `v1.18.1`

Reply:
I chose v1.15.0 as that is the version that the logger framework in controller-runtime is pulling. Matching it will keep surprises to minimum. We only need the import to get the Options, functionality is still handled by the controller runtime.